lib/reek/reek_exception.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module Reek
+  #
+  # @public
+  #
+  # ReekException is the only exception that Reek should give back
+  # to the user in case something goes wrong.
+  class ReekException < RuntimeError
```

Comment:
Any reason you choose repeating `Reek` (`Reek::ReekException`) instead of something more concise like `Reek::Error`?